packages/core/src/test/amazonq/common/contentController.test.ts

```diff
@@ -0,0 +1,44 @@
+import * as vscode from 'vscode'
+import assert from 'assert'
+import { EditorContentController } from '../../../amazonq/commons/controllers/contentController'
+import { openATextEditorWithText } from '../../testUtil'
+
+describe('contentController', () => {
+    let controller: EditorContentController
+
+    beforeEach(async () => {
+        controller = new EditorContentController()
+    })
+
+    afterEach(async function () {})
+
+    describe('insertTextAtCursorPosition', () => {
+        it('insert code when left hand size has no non empty character', async () => {
+            const editor = await openATextEditorWithText('def hello_world():\n    ', 'test.py')
+            if (editor) {
```

Comment:
if editor isn't found for some reason can we fail the test, since we won't actually know if function execution happened successfully